### ISSUE TITLE
Q about creating an external application using Cmake in VS2012

### ISSUE THREAD

**Issue by ioflowstudios** [uic-evl/omegalib#111]
Hi Guys,

When I'm learning new stuff in C++ I normally just make one solution with multiple projects, i.e.:
- Solution : SomeLibrary_Testing
  - Project : HelloWorld1
  - Project : HelloWorld2
  - Project : SomethingAbitHarder
  - Project : ShootingInTheDark

On your NewApplication wiki page you note that the application object created in main takes one string parameter and that it must be the same as the application name or the app won't work when run in cluster mode.

When you say application name, do you mean the name of the c++ file the main method is in, the project folder that the c++ file is in, or the entire solution as named in the example CMake file in where you say project(myapp)?

In other words, should I only have one project per solution?
